ui/src/adapters/api/connections.ts

```diff
@@ -0,0 +1,87 @@
+import axios from "axios";
+import { z } from "zod";
+
+import { APIResponse, HTTPStatusSuccess, ResultOK, buildAPIError } from "src/adapters/api";
+import { Env } from "src/domain";
+import { API_VERSION, QUERY_SEARCH_PARAM } from "src/utils/constants";
+import { StrictSchema } from "src/utils/types";
+
+const buildAuthorizationHeader = (env: Env) =>
+  `Basic ${window.btoa(`${env.api.username}:${env.api.password}`)}`;
+
+//TODO move it to credentials when is properly cleaned
```

Comment:
What needs cleaning? Is there an issue associated with this?

Reply:
We are keeping things there to doesn't break the Credentials section, so I thought to wait until we start that section

Reply:
We will be adding things to credentials.ts anyway, so it won't break anything. Better to add it where it needs to go instead of potentially forgetting it here.

Reply:
uff I don't know when `oldCredentials` is going to be refactored, so I've renamed and moved the new ones there